install_config/registry/extended_registry_configuration.adoc

```diff
@@ -80,42 +80,58 @@ We recommend against rebooting the entire cluster because that incurs
+== Configuring an External Registry Search List
+
+You can use the *_/etc/containers/registries.conf_* file to create a list of 
+Docker registries to search for container images. 
+
+The *_/etc/containers/registries.conf_* file is a list of registry servers 
+that {product-title} should search against when a user pulls an image using
+the image short name. For example, if the registry search list contains 
+`registry.redhat.io`, `docker.io`, and `registry.example.com`, and the 
+image short name is `myimage:latest`, {product-title} searches 
+the registries in that order until it finds the `myimage:latest`. 
+
+The *_/etc/containers/registries.conf_* file allows you to customize the 
+order of the search, specify secure and insecure registries, and define a blocked
+registry list. 
+
+{product-title} can pull images from any secure registry, which is a registry 
+that uses SSL/TLS, or insecure registries, which require standard web protocols 
+instead of TLS. {product-title} does not search or allow pulls from 
+registries on the blocked list and returns an error message stating 
+that the registry is not allowed.
+
+The registry search list allows you to curate a set of images and templates 
+that are available for download by {product-title} users. You can place these 
+images in one or more Docker registries, then add the registry 
+to the list, which you can pull into your cluster.
+
+When using the registry search list, {product-title} pulls only 
+from the specified registries. If users try to pull from a registry that 
+is not in the search list, they will receive an error message stating that 
+this registry is not allowed.
 
-To configure a whitelist:
+To configure a registry search list:
 
-. Edit the *_/etc/sysconfig/docker_* file to block all registries:
+. Edit the *_/etc/containers/registries.conf_* file to add or edit the following
+parameters as needed:
 +
 ----
-BLOCK_REGISTRY='--block-registry=all'
-----
-+
-You may need to uncomment the `*BLOCK_REGISTRY*` line.
-. In the same file, add registries to which you want to allow access:
-+
-----
-ADD_REGISTRY='--add-registry=<registry1> --add-registry=<registry2>'
-----
-+
-.Allowing Access to Registries
-----
-ADD_REGISTRY='--add-registry=registry.redhat.io'
-----
-+
-This example would restrict access to images available on the
-link:https://access.redhat.com/search/#/container-images[Red Hat Customer Portal].
+[registries.search] <1>
+registries = ["reg1.example.com", "reg2.example.com"]
 
-Once the whitelist is configured, if a user tries to pull from a container image registry
-that is not on the whitelist, they will receive an error message stating that
-this registry is not allowed.
+[registries.insecure] <2>
+registries = ["reg3.example.com"] 
 
+[registries.block] <3>
+registries = ['docker.io']
+----
+<1> Specify the secure registries from which users can download images 
+using SSL/TLS.
+<2> Specify the insecure registries from which users can download images 
```

Comment:
I think this is just _HTTP_.

Reply:
Or it could be _without_ TLS; because actually HTTP is always used; It happens either over TLS/SSL or not. The underlying protocol is always HTTP.